manual/manual/cmds/intf-c.etex

```diff
@@ -1712,7 +1712,8 @@ information is available, but the backtrace mechanism needs to be
 turned on programmatically.   This can be achieved from the OCaml side
 by calling "Printexc.record_backtrace true" in the initialization of
 one of the OCaml modules.  This can also be achieved from the C side
-by calling "caml_record_backtrace(Val_int(1));" in the OCaml-C glue code.
+by calling "caml_record_backtraces();" in the OCaml-C glue code.
+("ocaml_receord_backtraces" is declared in "backtrace.h")
```

Comment:
Typo "receord"

Reply:
Yuck, how many typos can one fit in one added sentence :-/ Fixed (and the `ocaml_` bit too!)